<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1197</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>921</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnquery">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>展示所有数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnInDB">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>导入数据库</string>
    </property>
   </widget>
   <widget class="QLabel" name="salary_low">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>20</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>薪资下限：</string>
    </property>
   </widget>
   <widget class="QLabel" name="salary_up">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>60</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>薪资上限：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="salary_lower">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="salary_upper">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalaryQuery">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>100</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>薪资查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExpQuery">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>280</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>年限查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNameQuery">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>职位查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="job_label">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>420</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>职位：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="jobName">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>410</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="URL_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>URL：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="URL_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cookie_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>cookie:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cookie_lineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnspider">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>爬取数据</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSalarySort">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>140</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>薪资上限降序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExpSort">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>320</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>年限下限升序</string>
    </property>
   </widget>
   <widget class="QLabel" name="exp_up">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>240</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>年限上限：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exp_upper">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="exp_low">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>200</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>年限下限：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="exp_lower">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExpSalaryQuery">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>370</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>薪资和年限共同查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="skill_label">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>510</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>技能：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="skill">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>500</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkillQuery">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>技能查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="welfare_label">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>610</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>福利</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="welfare">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>600</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWelfareQuery">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>650</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>福利查询</string>
    </property>
   </widget>
   <zorder>tableWidget</zorder>
   <zorder>btnquery</zorder>
   <zorder>btnInDB</zorder>
   <zorder>salary_low</zorder>
   <zorder>salary_up</zorder>
   <zorder>btnSalaryQuery</zorder>
   <zorder>btnExpQuery</zorder>
   <zorder>btnNameQuery</zorder>
   <zorder>job_label</zorder>
   <zorder>jobName</zorder>
   <zorder>salary_lower</zorder>
   <zorder>salary_upper</zorder>
   <zorder>URL_label</zorder>
   <zorder>URL_lineEdit</zorder>
   <zorder>cookie_label</zorder>
   <zorder>cookie_lineEdit</zorder>
   <zorder>btnspider</zorder>
   <zorder>btnSalarySort</zorder>
   <zorder>btnExpSort</zorder>
   <zorder>exp_up</zorder>
   <zorder>exp_upper</zorder>
   <zorder>exp_low</zorder>
   <zorder>exp_lower</zorder>
   <zorder>btnExpSalaryQuery</zorder>
   <zorder>skill_label</zorder>
   <zorder>skill</zorder>
   <zorder>btnSkillQuery</zorder>
   <zorder>welfare_label</zorder>
   <zorder>welfare</zorder>
   <zorder>btnWelfareQuery</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1197</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
